src/walter/walter_command.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+""" Run WALTer simulation as a command
+"""
+import argparse
+from subprocess import Popen
+import pandas as pd
+
+from walter import project
+
+usage = """
+WALTer generates a project directory and run simulations inside this directory.
+
+
+1. To create a simulation project only, type:
+
+       walter -p simu_walter
+
+
+2. To run simulations inside the project, type:
+
+       walter -p simu_walter -i sim_scheme.csv
+
+    or, inside an existing project:
+
+       cd simu_walter
+       walter -i sim_scheme.csv
+
+3. To run project management only, without running te model (for debug)  
+
+       walter -i sim_scheme.csv -p simu_walter --dry_run 
+
+"""
+
+
+def walter_parser():
+    parser = argparse.ArgumentParser(description=usage)
+    parser.add_argument("-i", type=str,
+                        help="Select input simulation scheme")
+    parser.add_argument("-p", type=str,
+                        help="Name of the project where simulations will be run")
+    parser.add_argument("--dry_run", help="do only project management tasks", action="store_true")
+    return parser
+
+
+def main():
+
+    parser = walter_parser()
+    args = parser.parse_args()
+
+    p = '.'
+    if args.p:
+        p = args.p
+    else:
+        if not project.check_cwd():
+            answer = raw_input("Current directory doesn't look like a walter project dir, Continue ? [Y/N]")
+            if not answer == 'Y':
+                return
+
+    sim_scheme = ''
+    if args.i:
+        sim_scheme = args.i
+
+    prj = project.Project(p)
+
+    # TODO: add a flag in the project to know if the project has been generated, modified or not.
+    if prj.dirname.exists():
+        print('Use Project %s located at %s' % (prj.name, prj.dirname))
+    else:
+        print('Project %s has been generated at %s' % (prj.name, prj.dirname))
+
+    if args.i:
```

Comment:
I do not understand these two lines: if args.i then sim_scheme == args.i

Reply:
that was a mistake de to missunderstanding of parser.
 I put the correct logic in walter_parser, update command doc and add a test in test_command_line.py: 
-p only : do not run anything
-i : run walter without args (lpy defaults)
-i my_sim.csv;: run walter with parameters in my_sim.csv